pkg/controller/cyclenoderequest/transitioner/util.go

```diff
@@ -536,3 +536,38 @@ func (t *CycleNodeRequestTransitioner) validateInstanceState(validNodeGroupInsta
 
 	return false, nil
 }
+
+// deleteFailedSiblingCNRs finds the CNRs generated for the same nodegroup as
+// the one in the calling transitioner. It filters for deleted CNRs in the same
```

Comment:
nit: it's not a calling transitioner, it's the parent struct technically. I would just say "the one in the transitioner"